model/distro/db.go

```diff
@@ -23,6 +23,7 @@ var (
 	SpawnAllowedKey     = bsonutil.MustHaveTag(Distro{}, "SpawnAllowed")
 	ExpansionsKey       = bsonutil.MustHaveTag(Distro{}, "Expansions")
 	DisabledKey         = bsonutil.MustHaveTag(Distro{}, "Disabled")
+	MaxContainersKey    = bsonutil.MustHaveTag(Distro{}, "MaxContainers")
```

Comment:
This is fine currently, but don't forget to remove this field when you implement the container pools configuration